### DISCUSSION TITLE
I see it on vscode[ERROR] [default] [2022-04-10T04:56:49.981Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"

### DISCUSSION THREAD

**Discussion by lpy12123213** [community/community#14647] [Copilot Conversations]
I cannot use GitHub copilot in vscode, and [error] [default] [2022-04-10t04:56:49.981z] GitHub copilot could not connect to server is displayed Extension activation failed: "unable to verify the first certificate"。 However, I can use it in pycharm
![1](https://user-images.githubusercontent.com/98199090/162602320-436ac457-72a9-411d-af92-64255ab84f85.png)
![2](https://user-images.githubusercontent.com/98199090/162602362-32323cdd-1352-4d74-baab-16c40fefde0d.png)

**Comment by c-0ne** [comment]
have the exactly same problem.
did you found anything to resolve this?

**Reply by renatodru** [reply]
The problem is about trust in chain certificate.
The solution is:
1 - install the extension in VSC "win-ca"
2 - change config "Win-ca: inject (Method to provide fetched root certificates to other externsions)" to **append**.
3 - restart VSC.
